Modules/Loadable/Segmentations/qSlicerSegmentationsSettingsPanel.cxx

```diff
@@ -94,7 +94,7 @@ void qSlicerSegmentationsSettingsPanelPrivate::init()
                       "Enable closed surface representation smoothing by default", ctkSettingsPanel::OptionNone);
   q->registerProperty("Segmentations/DefaultTerminologyEntry", q,
                       "defaultTerminologyEntry", SIGNAL(defaultTerminologyEntryChanged(QString)),
-                      "Defult terminology entry", ctkSettingsPanel::OptionNone);
+                      "Default terminology entry", ctkSettingsPanel::OptionNone);
```

Comment:
This updated `settingLabel` should be ok?

Reply:
Yes, it is OK, it is just text displayed to the user.